lib/fog/compute/google/models/instance_group.rb

```diff
@@ -27,9 +27,39 @@ def destroy(_async = true)
           data = service.delete_instance_group(name, zone_name)
         end
 
+        def add_instances(instances)
+          requires :identity, :zone
+
+          service.add_instance_group_instances(
+            identity, zone_name, format_instance_list(instances)
+          )
+        end
+
+        def remove_instances(instances)
+          requires :identity, :zone
+
+          service.remove_instance_group_instances(
+            identity, zone_name, format_instance_list(instances)
+          )
+        end
+
+        def list_instances
```

Comment:
This will return:
```
=> [{"instance"=>"https://www.googleapis.com/compute/v1/projects/dev-utility-90115/zones/europe-west1-d/instances/chef-server", "status"=>"RUNNING"},
 {"instance"=>"https://www.googleapis.com/compute/v1/projects/dev-utility-90115/zones/europe-west1-d/instances/elk-server", "status"=>"RUNNING"}]
```
@plribeiro3000 - PTAL - is such a return acceptable or should I return an array of self-links?
Additionally, if there's no data it passes back nil - is that ok or should it be empty array?